tests/cpp/container_test.cc

```diff
@@ -310,7 +310,7 @@ TEST(Map, Mutate) {
   CHECK(it != dict.end() && (*it).second.same_as(x));
 
   it = dict2.find(zz);
```

Comment:
Make sure the Map test covers the DenseMap and SmallMap range, as well as transition from Small to Dense